src/main/java/com/google/ehub/servlets/ItemSubmissionServlet.java

```diff
@@ -18,23 +18,45 @@
   private static final String TITLE_PARAMETER_KEY = "Title";
   private static final String DESCRIPTION_PARAMETER_KEY = "Plot";
   private static final String IMAGE_URL_PARAMETER_KEY = "Poster";
+  private static final String RELEASE_DATE_PARAMETER_KEY = "Released";
+  private static final String RUNTIME_PARAMETER_KEY = "Runtime";
+  private static final String GENRE_PARAMETER_KEY = "Genre";
+  private static final String DIRECTORS_PARAMETER_KEY = "Director";
+  private static final String WRITERS_PARAMETER_KEY = "Writer";
+  private static final String ACTORS_PARAMETER_KEY = "Actors";
 
   private static final int MAX_TITLE_CHARS = 150;
-  private static final int MAX_DESCRIPTION_CHARS = 500;
+  private static final int MAX_CHARS = 500;
```

Comment:
add documentation here or below on where this applies to

Reply:
I added documentation in last commit explaining what the 'MAX_CHARS' variable is referring to.